tests/test_openapi_parser/test_properties.py

```diff
@@ -24,13 +24,23 @@ def test_get_type_string(self):
         p = Property(name="test", required=True, default=None, nullable=False)
```

Comment:
Is there any way we can have a more end-to-end test that unset values are not sent over to the API endpoint?

I'm a bit nervous since I'm very unfamiliar with this codebase.